serrano/resources/preview.py

```diff
@@ -133,21 +151,15 @@ def get(self, request):
         model_name = opts.verbose_name.format()
         model_name_plural = opts.verbose_name_plural.format()
 
-        data = self.get_page_response(request, paginator, page)
-
-        data.update({
+        data = {
             'keys': header,
             'items': objects,
             'item_name': model_name,
             'item_name_plural': model_name_plural,
-            'item_count': paginator.count
-        })
-
-        path = reverse('serrano:data:preview')
-        links = self.get_page_links(request, path, page, extra=params)
```

Comment:
I think it is worth returning, otherwise the client would *have* to count to get the links. It is much cheaper to send a request that returns 0 data than performing a count.